cli-tests/src/upload.rs

```diff
@@ -1059,9 +1079,11 @@ async fn test_uncloned_repo_requires_manual_settings() {
     let state = MockServerBuilder::new().spawn_mock_server().await;
 
     let assert = CommandBuilder::upload(temp_dir.path(), state.host.clone())
+        .bazel_bep_path(bep_path.to_str().unwrap())
         .use_uncloned_repo(true)
         .command()
         .assert()
+        .code(predicate::eq(2))
```

Comment:
Asserting only on `failure` is bad practice since we aren't checking what the `failure` is for.

Reply:
`exitcode` doesn't define 2 as a constant:
https://github.com/benwilber/exitcode/blob/5d18ab34e680180b0c032707c5a42592a075af95/src/lib.rs

This code comes directly from `clap` and isn't publicly accessible:
https://github.com/clap-rs/clap/blob/d117f7acdeedebaf5fd7847debb15c834423f159/clap_builder/src/util/mod.rs#L25-L30